static/src/stylesheets/inline/article-labour-liverpool.scss

```diff
@@ -259,3 +259,70 @@ $labour-liverpool-header-element-opacity: .85;
         height: 88%;
     }
 }
+.content__article-body {
+    figure {
+        &.element-video--heroic {
```

Comment:
Is there any way to reduce this level of nesting? Would it be possible to just target the `.element-video--heroic` directly, without `.content__article-body` and `figure`?